keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -521,6 +522,20 @@ Job controller. For example, Kubelet could restart a failed container before the
 Job controller decides to terminate the corresponding job due to a rule using
 `onExitCodes`.
 
+#### The scope of the FailureTarget condition
+
+As part of this KEP we introduced the
+[FailureTarget condition](#interim-failuretarget-job-condition) scoped to the
+failures due to pod failure policy.
+
+However, we are going to extend the scope of the condition to all Job failure
+scenarios (covering also backoffLimit exceeded and ActiveDeadlineSeconds
+exceeded), as part of fixing
+(issue #123775)[https://github.com/kubernetes/kubernetes/issues/123775].
+
+See more details in the
+[Job API managed-by mechanism](https://github.com/kubernetes/enhancements/issues/4368).
```

Comment:
Can you just point to the document itself, rather than the KEP issue? Iow. https://github.com/kubernetes/enhancements/blob/master/keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

Reply:
Done